spec/rubocop/cop/performance/inefficient_hash_search_spec.rb

```diff
@@ -47,6 +47,15 @@
       RUBY
     end
 
+    it 'does not register an offense when `keys` method defined by itself ' \
+       'and `include?` method are method chaining' do
```

Comment:
You'll have to update this in the new project as well, to keep in sync with this one.

Reply:
Sure. I plan to port to rubocop-performance repo.

Reply:
I am already doing some work on rubocop-performance today, so I added the spec in https://github.com/rubocop-hq/rubocop-performance/pull/12. I haven't imported the cops yet, but will make sure I get the latest version when I do.